### PR TITLE
Add ability to scale recipes - Issue #17

### PR DESCRIPTION
**Pull request by sruduke**
* Refactored logic in MealPlan to facilitate methods and fetching data
* Configured views and logic for scaling
* Altered adapters and listeners for real-time feedback

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -72,7 +72,6 @@
         <activity
             android:name=".recipe.RecipeDetailsActivity"
             android:exported="true" />
-        <activity android:name=".recipe.RecipeStorageActivity" />
```
Comment (jeastgaard) [inline app/src/main/AndroidManifest.xml]: Why is the storage activity removed from here?

File: app/src/main/java/com/example/happymeals/mealplan/MealPlanStorage.java [hunk 2]
```diff
@@ -138,8 +127,9 @@ public ArrayList< MealPlan > getMealPlans() {
      */
     public void addMealPlan( MealPlan mealPlan ) {
         mealPlans.add( mealPlan );
+        System.out.println(mealPlans.toString());
```
Comment (jeastgaard) [inline app/src/main/java/com/example/happymeals/mealplan/MealPlanStorage.java]: Debug code

**Review by jeastgaard — CHANGES_REQUESTED** [review]
Small debug code and question

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (sruduke) [reply app/src/main/AndroidManifest.xml]: Fixed in latest commit

File: app/src/main/java/com/example/happymeals/mealplan/MealPlanStorage.java (hunk 2 above)
Comment (sruduke) [reply app/src/main/java/com/example/happymeals/mealplan/MealPlanStorage.java]: Right, good catch

**Review by jeastgaard — APPROVED** [review]
Small debug code and question

**Pull request merged**